stylesheets/main.css

```diff
@@ -150,3 +150,12 @@ div.menu nav ul {
     z-index: 0; /* menu above sections */
   }
 }
+
+@media (max-width: 400px) {
```

Comment:
Why 400px?

Reply:
This was arbitrary, I wanted something that was good enough for most mobile devices in portrait, so that there wouldn't be too much spacing above the heading when jumping to a section.

After a bit of research, I realize that some larger phones are in the 400-500 range so I can bump up this value.

Reply:
I suggest putting it in the `max-width: 799px` block above unless that's too wide a range.

Reply:
@aakside - I set the spacing to 40px in the `max-width: 799px` block already, wanted to make it smaller for 400px and under.

Reply:
Here are some screenshots to illustrate what I mean.

10px padding on mobile:
![image](https://user-images.githubusercontent.com/8042780/77610220-7ad5d400-6ef8-11ea-9d64-795ee7c88ba4.png)

40px padding on mobile (gap between nav and header is a bit big):
![image](https://user-images.githubusercontent.com/8042780/77610258-9345ee80-6ef8-11ea-9fc0-02bdc648b35d.png)

10px padding on tablet (heading is hidden underneath nav)
![image](https://user-images.githubusercontent.com/8042780/77610295-ac4e9f80-6ef8-11ea-83e3-84e16b4b6ade.png)

40px padding on tablet:
![image](https://user-images.githubusercontent.com/8042780/77610307-b7a1cb00-6ef8-11ea-87a2-f282af04a519.png)